dev-tools/startup/init-db.sql

```diff
@@ -0,0 +1,7 @@
+CREATE DATABASE discovery_provider_1;
+
+CREATE DATABASE creator_node_1;
+CREATE DATABASE creator_node_2;
+CREATE DATABASE creator_node_3;
+
+CREATE DATABASE identity_service;
```

Comment:
@theoilie covered the same stuff that the /etc/hosts change targets for now and is why I think we should isolate both into files for consistency